frontend/src/components/Inquiries/InquiriesTable.tsx

```diff
@@ -56,49 +68,131 @@ const InquiriesTable = () => {
     })
   }, [inquiries])
 
+  const queryClient = useQueryClient()
+  const toast = useToast()
+
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  const [selectedInquiry, setSelectedInquiry] = useState<InquiryPublic | null>(
+    null,
+  )
+  const [editText, setEditText] = useState("")
+
+  const updateInquiryMutation = useMutation({
+    mutationFn: async (updatedInquiry: InquiryPublic) => {
+      return await InquiriesService.updateInquiry(updatedInquiry)
```

Comment:
Yeah I'm not quite sure why we're calling InquiriesService from here and also from the Add/EditModal?

Reply:
It seems like we should just use the AddOrEditInquiryModal from here to do the actual editing. Then all the table needs to do is import it and call it. Right?